src/wp-includes/theme.php

```diff
@@ -157,7 +157,10 @@ function wp_clean_themes_cache( $clear_update_cache = true ) {
  * @return bool True if a child theme is in use, false otherwise.
  */
 function is_child_theme() {
-	return ( TEMPLATEPATH !== STYLESHEETPATH );
+	$stylesheet_path = get_stylesheet_directory();
+	$template_path   = get_template_directory();
+
+	return ( $template_path !== $stylesheet_path );
```

Comment:
```suggestion
	return ( get_template_directory() !== get_stylesheet_directory() );
```
I noticed that you have two variables, `$stylesheet_path` and `$template_path`, which are only used once in the code. Given that they are used only once, we can simplify the code by directly comparing the functions `get_template_directory()` and `get_stylesheet_directory()`. This will make the code more concise and easier to read.

However, if there are other purposes for these variables that I'm not aware of, please let me know.

Reply:
Addressed in https://github.com/WordPress/wordpress-develop/pull/5160/commits/a2b557418c4a47dc776dedc246eb8473bf23ea8f